content/en/security/application_security/threats/add-user-info.md

```diff
@@ -21,6 +21,10 @@ Instrument your services and track user activity to detect and block bad actors.
 
 [Track user logins and activity](#adding-business-logic-information-login-success-login-failure-any-business-logic-to-traces) to detect account takeovers and business logic abuse with out-of-the-box detection rules, and to ultimately block attackers.
 
+<div class="alert alert-info">
+<strong>Note: Automated Detection of User Activity</strong> - when possible, Datadog Tracing Libraries will attempt to detect and report user activity events automatically. For more information, read <a href="/security/application_security/threats/add-user-info/?tab=set_user#disabling-automatic-user-activity-event-tracking">Disabling automatic user activity event tracking</a>.
```

Comment:
Did we settle on `Automated Detection of User Activity`? 

What about `Automated User Activity Detection`?

Reply:
Both the same for me. I defer to the docs team review on details like that.